build.py

```diff
@@ -23,6 +23,7 @@ def parse_arg():
   parser.add_option("-c", "--clean", action="store_true", dest="clean", default=False, help="clean")
   parser.add_option("-t", "--test", action="store_true", dest="test", default=False, help="clean")
   parser.add_option("-v", "--val", action="store_true", dest="val", default=False, help="build version used for gpu validation")
+  parser.add_option("--newpin", action="store_true", dest="pin_3_7_trace", default=False, help="trace based on pin 3.7 ")
```

Comment:
change 3.7 to appropriate version which i think is 3.13?

Reply:
resolved